src/source-map.ts

```diff
@@ -1,17 +1,43 @@
-import { AnyMap, originalPositionFor } from '@jridgewell/trace-mapping';
+import remapping from '@ampproject/remapping';
+import resolveUri from '@jridgewell/resolve-uri';
+import {
+  AnyMap,
+  originalPositionFor,
+  generatedPositionFor,
+  eachMapping,
+  encodedMappings,
+  sourceContentFor,
+} from '@jridgewell/trace-mapping';
 import {
   GenMapping,
   maybeAddMapping,
   toDecodedMap,
   toEncodedMap,
   setSourceContent,
+  fromMap,
 } from '@jridgewell/gen-mapping';
 
-import type { TraceMap, SectionedSourceMapInput } from '@jridgewell/trace-mapping';
-export type { TraceMap, SectionedSourceMapInput };
+import type {
+  TraceMap,
+  SourceMapInput,
+  SectionedSourceMapInput,
+  DecodedSourceMap,
+} from '@jridgewell/trace-mapping';
+export type { TraceMap, SourceMapInput, SectionedSourceMapInput, DecodedSourceMap };
+
+import type { Mapping, EncodedSourceMap } from '@jridgewell/gen-mapping';
+export type { Mapping, EncodedSourceMap };
+
+import { join, relative } from './util';
 
-import type { Mapping, EncodedSourceMap, DecodedSourceMap } from '@jridgewell/gen-mapping';
-export type { Mapping, EncodedSourceMap, DecodedSourceMap };
+function resolve(input: string, base: string | undefined): string {
```

Comment:
I took this from https://github.com/jridgewell/trace-mapping/blob/main/src/resolve.ts

Reply:
Removed